app/labor/cache_buster.rb

```diff
@@ -206,3 +239,12 @@ def self.bust_article_comment(commentable)
     end
   end
 end
+
+# Creates our own purge method for an HTTP request,
+# which is used by Nginx to bust a cache.
+# See Net::HTTPGenericRequest for attributes/methods.
+class Net::HTTP::NginxPurge < Net::HTTPRequest # rubocop:disable Style/ClassAndModuleChildren
```

Comment:
is there a reason why it's called `NginxPurge` and not just Purge? after all the HTTP method is `PURGE` and this can become a generic extension to be used anywhere that method is supported